src/reports/make-graph.rkt

```diff
@@ -89,9 +89,6 @@
                     (/ start-cost cost))])
       (and (not (null? better)) (apply max better))))
 
-  (match-define (list train-pctx test-pctx) pctxs)
-
-  (define end-alt (car end-alts))
```

Comment:
Unused?

Reply:
Yep. left over and could have cleaned up in the server PR.